manager/kafka/kafka.go

```diff
@@ -0,0 +1,75 @@
+package kafka
+
+import (
+	"app/base"
+	"app/base/database"
+	"app/base/models"
+	"app/base/mqueue"
+	"app/base/utils"
+)
+
+var (
+	evalWriter               mqueue.Writer
+	baselinesChannel         chan baselineAndAccount
+	enableEvaluationRequests = utils.GetBoolEnvOrDefault("ENABLE_EVALUATION_REQUESTS", true)
+)
+
+type baselineAndAccount struct {
+	BaselineID *int
+	AccountID  int
+}
+
+func TryStartEvalQueue(createWriter mqueue.CreateWriter) {
+	if !enableEvaluationRequests {
+		return
+	}
+	evalTopic := utils.GetenvOrFail("EVAL_TOPIC")
+	evalWriter = createWriter(evalTopic)
+	baselinesChannel = make(chan baselineAndAccount)
+	go runBaselineRecalcLoop()
+}
+
+func runBaselineRecalcLoop() {
+	for {
+		baseline := <-baselinesChannel
+		inventoryIDs := getInventoryIDs(baseline.BaselineID, baseline.AccountID)
+		sendInventoryIDs(inventoryIDs)
+	}
+}
+
+func getInventoryIDs(baselineID *int, rhAccountID int) []mqueue.InventoryAID {
+	var inventoryAIDs []mqueue.InventoryAID
+	err := database.Db.Model(&models.SystemPlatform{}).
+		Select("inventory_id, rh_account_id").
+		Where(map[string]interface{}{"rh_account_id": rhAccountID, "baseline_id": baselineID}).
```

Comment:
Are you sure that in case baseline_id is nil this will translate into "baseline_id IS NULL" not "baseline = NULL"?

Reply:
Yes, I wrote a test for that.